core/senders/qq_batch_builder.py

```
@@ -133,9 +164,17 @@ async def build_processed_batches(
                 media_components = []
                 has_any_attachment = False
                 msg_max_size = getattr(msg, "_max_file_size", 0)
+                expects_media = message_expects_downloadable_media(msg)
                 files = await sender.downloader.download_media(
                     msg, max_size_mb=msg_max_size
                 )
+                if expects_media and not files:
+                    media_download_failed = True
+                    logger.warning(
+                        f"[QQSender] 消息 {getattr(msg, 'id', '?')} 媒体下载失败，"
```

Ruff (0.16.0)

[warning] String contains ambiguous `，` (FULLWIDTH COMMA). Did you mean `,` (COMMA)?

(RUF001)